discovery-provider/src/api/v1/tracks.py

```diff
@@ -804,3 +809,172 @@ def get(self, track_id):
         tracks = get_remix_track_parents(args)
         tracks = list(map(extend_track, tracks))
         return success_response(tracks)
+
+
+"""
+  Gets a windowed (over a certain timerange) view into the "top" of a certain type
+  amongst followees. Requires an account.
+  This endpoint is useful in generating views like:
+      - New releases
+
+  Args:
+      window: (string) The window from now() to look back over. Supports  all standard SqlAlchemy interval notation (week, month, year, etc.).
+      limit?: (number) default=25, max=100
+"""
+best_new_releases_parser = reqparse.RequestParser()
+best_new_releases_parser.add_argument("window", required=True, type=str)
+best_new_releases_parser.add_argument("limit", required=False, default=25, type=int)
+best_new_releases_parser.add_argument("with_users", required=False, default=True, type=bool)
+
+
+@full_ns.route("/best_new_releases")
+class BestNewReleases(Resource):
+    @record_metrics
+    @cache(ttl_sec=10)
+    @full_ns.marshal_with(full_tracks_response)
+    def get(self):
+        request_args = best_new_releases_parser.parse_args()
+        
+        window = request_args.get("window")
+        valid_windows = ["week", "month", "year"]
+        if not window or window not in valid_windows:
+            abort_bad_request_param("window", full_ns)
+        
+        args = {
+            "with_users": request_args.get("with_users"),
+            "limit": format_limit(request_args, 100, 25),
+        }
+        
+        tracks = get_top_followee_windowed("track", window, args)
```

Comment:
For this method and the latter methods `get_feed`, `get_top_followee_saves`, and this query `get_top_followee_windowed` -  they call `get_current_user_id` from the sr/query_helpers file which pulls the user id out of the request headers "X-User-ID" file. 
Since we migrated to the /v1 endpoints, we not pass in the user_id as an encoded  query param. 
Could you update the queries so that current_user_id is passed in as a param to these methods and update the old routes to call the old get_current_user_id directly?
Let me know if this doesn't make sense - happy to chat offline